validation/src/test/scala/hmda/validation/rules/ts/validity/V155Spec.scala

```diff
@@ -20,8 +20,9 @@ class V155Spec extends PropSpec with PropertyChecks with MustMatchers with TsGen
   property("Invalid emails should be detected") {
     forAll(tsGen) { ts =>
       val badContact = badContactGen.sample.getOrElse(Contact("", "", "", ""))
+      println(badContact)
       val badTs = ts.copy(contact = badContact)
-      whenever(badTs.contact.email != "") {
+      whenever(badTs.id == 1) {
```

Comment:
I don't understand the purpose of having a `whenever` here at all. It seems like extra code to read, that doesn't affect the behavior of the test. What am I missing?